modules/infra/control/graph.c

```diff
@@ -338,6 +338,7 @@ static void graph_fini(struct event_base *) {
 static struct gr_module graph_module = {
 	.name = "graph",
 	.init = graph_init,
+	.init_prio = -999,
```

Comment:
Could you define a set of enum or #define for the prios instead of hardcoded values ? I'll help to grep the code.

Reply:
We don't use constants for any of these priorities. I don't think it would bring much value since the same priority should not be used more than once.